src/classes/ADDR_Seasonal_TEST.cls

```diff
@@ -902,7 +902,7 @@ private with sharing class ADDR_Seasonal_TEST {
             MailingState__c = 'Texas',
             MailingCountry__c = 'United States',
             Seasonal_Start_Day__c = String.valueOf(system.today().addDays(-1).day()),
-            Seasonal_Start_Month__c = String.valueOf(system.today().addMonths(-1).month()),
+            Seasonal_Start_Month__c = String.valueOf(system.today().month()),
```

Comment:
This is a good suggestion. It looks like my previous pull request did not take January into consideration. However, this change will still leave an error condition on the 1st of each month.